### PR TITLE
defaultstorageclass-protection annotation changes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

chore

**What this PR does/ why we need it**:

minor changes to annotations

**Special notes for your reviewer**:
This has no material effect on the function of this addon

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: defaultstorageclass-protection
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.5-2"
-    appversion.kubeaddons.mesosphere.io/defaultstorageclass-protection: "0.0.5"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-0"
```
Comment (samvantran) [inline addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml:8]: i'm wondering why 1.1.0 and not 1.0.0 but okay, works for me!

**Review by samvantran — APPROVED** [review]

File: addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml (hunk 1 above)
Comment (joejulian) [reply addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml:8]: No real reason.. Sort of it's not new so I felt like 1.0.0 was wrong.

**Review by branden — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.3

**Pull request merged**